### PR TITLE
fix pathFilter for alt browser field

### PR DESCRIPTION
**Pull request by mvayngrib**
oops, missed this one the first time around. Browserify's tests unearthed it

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -180,17 +180,17 @@ function build_resolve_opts(opts, base) {
     	if(mappedPath) {
     		return mappedPath;
     	}
-    	if(!info.browser) {
+
+        var replacements = info[browser];
```
Comment (defunctzombie) [inline index.js]: syntax isssue

File: index.js [hunk 2]
```diff
@@ -180,17 +180,17 @@ function build_resolve_opts(opts, base) {
...
 
-    	if(typeof info.browser) {
-    		mappedPath = info.browser[relativePath];
-    		if(!mappedPath && (relativePath.lastIndexOf(".js") === relativePath.length-3) ) {
-    			mappedPath = info.browser[relativePath+".js"];
-    		}
-    		return mappedPath;
-    	}
+		mappedPath = replacements[relativePath];
+		if(!mappedPath && (relativePath.lastIndexOf(".js") === relativePath.length-3) ) {
```
Comment (defunctzombie) [inline index.js]: single quotes for strings

File: index.js [hunk 3]
```diff
@@ -180,17 +180,17 @@ function build_resolve_opts(opts, base) {
...
+			mappedPath = replacements[relativePath+".js"];
```
Comment (defunctzombie) [inline index.js]: single quotes for strings, space around `+` operator

**Comment by defunctzombie** [comment]
Please add tests so it is not broken again; then I will merge.

File: index.js (hunk 3 above)
Comment (mvayngrib) [inline index.js]: np, thought it was your style choice

Comment (defunctzombie) [inline index.js]: maybe an oversight on a previous PR or laziness on my part.

**Comment by mvayngrib** [comment]
added missing tests. "deep module reference mapping (alt-browser)" detects the overlooked pathFilter bug.

also some style fixes, and added .jscsrc, idk if you want that in here

**Comment by defunctzombie** [comment]
What's with all the changes to the test fixtures?

**Comment by mvayngrib** [comment]
added "chromeapp" fields to various package.json's, added files "chromeapp" fields pull in, to differentiate from the ones pulled in by "browser" fields

**Comment by mvayngrib** [comment]
i'm starting to think separate tests for an alt browser field are unnecessary, so I wouldn't merge this if I were you. In fact the previous tests I added should be scrapped too, as "browser" is now nothing more than a default.

**Comment by defunctzombie** [comment]
K, so can we make just a test for the issue this is fixing?

**Comment by mvayngrib** [comment]
added, my bad dragging this out

**Comment by defunctzombie** [comment]
Thanks for being vigilant and following up! Fix is out in 1.8.2

**Pull request merged**